### ISSUE TITLE
A subtle color for inline blame?

### ISSUE THREAD

**Issue by aaravind100** [Sertion/vscode-gitblame#145]
Hey @Sertion , Good day!!

Would a more subtle color for the inline blame make sense or maybe an option to set a custom color code? 

Presently the color is set by the `editorCodeLens.foreground` [here](https://github.com/Sertion/vscode-gitblame/blob/c3a9e04eb61a0dc8707004d6304d7c9334855d35/src/view.ts#L67). This looks good with the default theme, but looks prominent in some themes, rose pine for example.

- Rosé Pine Moon

![image](https://github.com/Sertion/vscode-gitblame/assets/98406957/5f77cf3f-6dc3-4e06-bb1f-c468ac1f98b6)

- Vscode Dark Modern

![image](https://github.com/Sertion/vscode-gitblame/assets/98406957/a5adf801-7f4a-4a6c-8391-6847975bdc10)

- Vscode Dark Modern with `editorWhitespace.foreground` maybe?

![image](https://github.com/Sertion/vscode-gitblame/assets/98406957/3c8afa60-f066-4ae7-a9ce-01bd12dfe4ce)

This is indeed trying to emulate the style from [gitlens](https://raw.githubusercontent.com/gitkraken/vscode-gitlens/main/images/docs/current-line-blame.png), which has become quite bloated lately :sweat_smile: 

I'd be happy to submit a pr for this.

```
vscode version: 1.79.2
extension version: v10.2.0
```

**Comment by Sertion** [comment]
Hey Ajith.

Chasing the perfect color for each theme is a Sisyphean task. `editorCodeLens.foreground` was chosen because it is the color used when showing information about the code in the editor. It should be possible to [override this color in your settings](https://code.visualstudio.com/docs/getstarted/themes#_customizing-a-color-theme).

**Comment by aaravind100** [comment]
I understand, thanks for the suggestions :smile:

**Comment by redactedscribe** [comment]
@Sertion Could it be possible to change which colour the inline blame references? E.g. to point it at the theme's comment colour variable instead? The comments are usually more subtle than the foreground text. Instead of hardcoding foreground vs comment colour, perhaps leave it up to the user to define which colour variable to use? Thanks for Git Blame!

**Comment by Sertion** [comment]
> The comments are usually more subtle than the foreground text. Instead of hardcoding foreground vs comment colour, perhaps leave it up to the user to define which colour variable to use? Thanks for Git Blame!

Sadly there is no good way of allowing a "color" as a setting and writing something to detect what random color-like string the extension has gotten through a setting is not something I'm interested in exploring.

**Comment by Sertion** [comment]
Implemented in #168.

**Comment by aaravind100** [comment]
Thank you!!

FYI for anyone reading this, we should add this to setting.json to use a custom color.
```json
    "workbench.colorCustomizations": {
        "gitblame.inlineMessage": "<hex color code>"
    }
```
